projects/gnu-tools/monthly-report-2020-12-14.md

```diff
@@ -0,0 +1,79 @@
+# Software TG monthly meeting - CORE-V GNU Tool Chain (December 2020)
+
+## Overview
+
+### Key activities
+
+* Added multiply accumulate and general ALU operations support to GNU Binutils incl. full testing - in OpenHW repos (Embecosm)
+* Added post-increment and register-indexed load/store support to GNU Binutils incl. full testing - not yet in OpenHW repos (UOB/ETHZ)
+* Added general ALU operations support to GNU GCC, not tested as awaiting hardware - not yet in OpenHW repos (Embecosm)
+* Started adding immediate branching instructions support to support to GNU Binutils (UOB/ETHZ)
+* Added new -march option: xcorevalu (supports the general ALU operations subset) and xcorevpostinc (supports the post-increment and register-indexed load/store operations subset) in binutils and xcorevalu to GCC (UOB/ETHZ and Embecosm)
+* Updated documentation to include all CORE-V extensions (found at: https://github.com/openhwgroup/core-v-docs/tree/master/cores/cv32e40p/user_manual)
+
+### Task status
+
+| Category    | Last Month | This Month | Delta    |
+| -----------:| ----------:| ----------:| --------:|
+| Unstarted   | 11         | 8          | -3       |
+| In progress | 3          | 3          | -        |
+| Complete    | 6          | 11         | +5       |
+
```

Comment:
I notice this table is "Last Month" then "This Month". The tables below are "This month", "Last month".